README.md

```diff
@@ -172,20 +172,19 @@ Check the [systemd-journal plugin of Netdata](https://github.com/netdata/netdata
+|          More Customizability           |                           Set default settings for all charts and views!                            |     soon      |                                               in progress                                                 |
+|          AWS Integratd billing          |       Run Netdata our your AWS instances and get your billing integrated on your AWS account.       |     soon      |                                               in progress                                                 |
+|        Alert Silence Manager R2         |           Improvements to the Alert Silencing Manager with recurring schedules and more!            |     soon      |                                               in progress                                                 |
+|                Okta SSO                 |        Faciliate the integration of Netdata into your organzations user management process.         |     soon      |                                               in progress                                                 |
+| Prometheus/OpenMetrics<br/>improvements |               Allow users to configure how metrics should be ingested and presented.                |     soon      |                                               in progress                                                 |
 |                Loki logs                |                     Another Logs integration, bring your Loki logs onto the UI!                     |     soon      |                                               in progress                                               |
-|              Homelab plan               |                    Unlimited Netdata plan targetted for homelabbers or students.                    |     soon      |                                               in progress                                               |
-|       Easy Custom<br/>Dashboards        |         Drag and drop charts to create custom dashboards on the fly, while troubleshooting!         |     soon      |                                               in progress                                               |
-|               UCUM Units                |                    Migrate all metrics to the Unified Code for Units of Measure.                    |     soon      |                                               in progress                                               |
-|            Click to Activate            |                          Configure Alerts and Data Collectors from the UI!                          |     soon      |                                               in progress                                               |
+|            Click to Activate            |                          Configure Alerts and Data Collectors from the UI!                          |     soon      | Beta release [v1.45](https://github.com/netdata/netdata/releases/tag/v1.45.0#v1450-dynamic-configuration) - in progress |
```

Comment:
What does "Click to Activate " mean?

Reply:
Activation of configs from the UI, guess it was the term Costa used when he introduced this the first time

Reply:
I'm not sure I understand. The column name is "What", I think that is a feature name. Click what/click where? Active configs - what is that? 

Should it be "Dynamic configuration"? cc @Ancairon

Reply:
I agree, let me change that